src/main/jte/view_event.jte

```diff
@@ -0,0 +1,55 @@
+@import edu.kit.hci.soli.domain.Booking
+@import edu.kit.hci.soli.dto.LoginStateModel
+@import edu.kit.hci.soli.domain.ShareRoomType
+@import edu.kit.hci.soli.template.JteContext
+@import edu.kit.hci.soli.template.PageSpec
+@import org.springframework.security.web.csrf.CsrfToken
+@import java.time.LocalDateTime
+@import java.util.Objects
+
+@param JteContext context
+@param LoginStateModel login
+@param Booking booking
+@param Long room
+@param LocalDateTime start
+@param LocalDateTime end
+@param LocalDateTime minimumTime
+@param LocalDateTime maximumTime
```

Comment:
many of these parameters are not used and redundant as well!

Reply:
Additionally, they could be reordered so that "system" parameters (context, lsm, csrf) are above the "view" parameter (booking) and separated with a blank line
(I made those names up on the spot, idk what the proper term is)